code/game/objects/machinery/computer/camera_advanced.dm

```diff
@@ -21,12 +22,16 @@
 
 /obj/machinery/computer/camera_advanced/Initialize(mapload)
 	. = ..()
-	off_action = new
-	jump_action = new
 	actions = list()
 	for(var/i in networks)
 		networks -= i
 		networks += lowertext(i)
+	if(SOM_CAMERA_NETWORK in networks)
+		parent_cameranet = GLOB.som_cameranet
+	else
+		parent_cameranet = GLOB.cameranet
```

Comment:
Leery about this, but given that networks seems to support arbitrary length (why would you want multiple?) it works.

Reply:
yea this else is the messiest part

Reply:
what would be a better option? I guess I could just have a cameranet var or something, but at that point tying it to the network seems better